purchase_request_procurement/models/procurement.py

```diff
@@ -4,6 +4,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
```

Comment:
use **from openerp.exceptions import Warning as UserError** to comply with CONTRIBUTING.md